app/models/container_deployment.rb

```diff
@@ -0,0 +1,10 @@
+class ContainerDeployment < ApplicationRecord
+  has_many :container_deployment_authentications
+  belongs_to :deployed_ems, :class_name => 'ExtManagementSystem', :inverse_of => :container_deployments
+  belongs_to :deployed_on_ems, :class_name => 'ExtManagementSystem', :inverse_of => :container_deployments
```

Comment:
Yea placing it twice on the same name doesn't seems to work :/
From the other side of the relation it works. :)